extensions/observability-devservices/testcontainers/src/main/java/io/quarkus/observability/testcontainers/PrometheusYamlFile.java

```diff
@@ -0,0 +1,79 @@
+package io.quarkus.observability.testcontainers;
+
+public class PrometheusYamlFile {
```

Comment:
I understand it's for type-safety, but I think we can move this into an utility method in `ContainerConfigUtil` or similar?

Reply:
Good point @gastaldi let me move it.

Reply:
Done!

Reply:
Sorry for not being clear, I meant having a static method, not moving the entire class 😅

Reply:
well @alesj already had a static class in there and htis class is REALLY only used by the LgtmContainer so it was a good idea to move it entirely 😄

Reply:
Yeah, my point is that it feels unnecessary to instantiate an object just to produce a String, given that the setters are never called

Reply:
Ahhh right i see what you mean now.

Reply:
Removed the class entirely now. its just a String.format.